{{cookiecutter.project_shortname}}/Pipfile

```diff
@@ -7,7 +7,9 @@ verify_ssl = true
 check-manifest = ">=0.25"
 
 [packages]
-invenio-app-rdm = {extras = ["{{ cookiecutter.database }}", "elasticsearch{{cookiecutter.elasticsearch}}"],version = ">=1.0.0a4"}
+invenio-app-rdm = {extras = ["{{ cookiecutter.database }}", "elasticsearch{{cookiecutter.elasticsearch}}"],version = "==1.0.0a4"}
+invenio-rdm-records = "==1.0.0a6"
+invenio-records-permissions = "==1.0.0a4"
```

Comment:
I needed to do a lot of thinking about this to end up agreeing to go with this as a convenience hack:

We pin exact versions, to make sure that the packages we develop and release are not used by installs until we have made an official release that will depend on those new packages (and the cycle beings anew).

`Pipfile.lock` is probably what we really want (and it's fine to commit it for an application) and/or a distinction between `0.X.0` releases (at end of month) and `0.X.0aY` releases during month. But for now this will do. 

The main issue here (why we would want `.lock` or different versioning scheme) is the potential for a conflict between the pinned versions and the dependent versions in `invenio-app-rdm`. We just have to be careful (famous last words).